keyboards/clawsome/hatchback/readme.md

```diff
@@ -0,0 +1,13 @@
+# SUV
+
+This is TKL keyboard that can be converted down into a 75% keyboard and a 13key macropad (see "luggage rack")
+
+- Keyboard Maintainer: [AAClawson](https://github.com/AlisGraveNil)
+- Hardware Supported: Hatchback, Elite-C (as TKL); Pro-Micro and nice!nano (as 75%)
+- Hardware Availability: In stock within the next month
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?